src/components/Differential/DifferentialSearch.jsx

```diff
@@ -1020,7 +1078,7 @@ class DifferentialSearch extends Component {
           >
             <Radio
               toggle
-              label={PlotLabel}
+              label="View Plot Intersections"
```

Comment:
"View Plot Intersections" is quite long...I've added a window width check to change CSS (drop the toggle to a new line) if the space is too tight.  I could maybe make it a listener so it's dynamic, but that might bog it down re-rendering

Reply:
If it drops, can it be immediately right justified as opposed to ...
![image](https://github.com/abbvie-external/OmicNavigatorWebApp/assets/8203433/7a63e67e-ecb2-4f5f-b571-488777a61e17)

Reply:
@ternst23, sure, I've added a resize listener so it will now change dynamically